l10n_ch_pain_credit_transfer/tests/test_ch_sct.py

```diff
@@ -6,331 +6,318 @@
-        self.assertEquals(len(bank_lines), 1)
+        self.assertEqual(len(bank_lines), 1)
         bank_line = bank_lines[0]
-        self.assertEquals(bank_line.currency_id, self.eur_currency)
-        self.assertEquals(bank_line.communication_type, "normal")
-        self.assertEquals(bank_line.communication, "Inv1242-Inv1248")
-        self.assertEquals(bank_line.partner_bank_id, invoice1.partner_bank_id)
+        self.assertEqual(bank_line.currency_id, self.eur_currency)
+        self.assertEqual(bank_line.communication_type, "normal")
+        self.assertEqual(bank_line.communication, "Inv1242-Inv1248")
+        self.assertEqual(bank_line.partner_bank_id, invoice1.partner_bank_id)
 
         action = self.payment_order.open2generated()
-        self.assertEquals(self.payment_order.state, "generated")
-        self.assertEquals(action["res_model"], "ir.attachment")
+        self.assertEqual(self.payment_order.state, "generated")
+        self.assertEqual(action["res_model"], "ir.attachment")
         attachment = self.attachment_model.browse(action["res_id"])
-        self.assertEquals(attachment.datas_fname[-4:], ".xml")
+        self.assertEqual(attachment.name[-4:], ".xml")
         xml_file = base64.b64decode(attachment.datas)
         xml_root = etree.fromstring(xml_file)
         # print "xml_file=", etree.tostring(xml_root, pretty_print=True)
         namespaces = xml_root.nsmap
         namespaces["p"] = xml_root.nsmap[None]
         namespaces.pop(None)
         pay_method_xpath = xml_root.xpath("//p:PmtInf/p:PmtMtd", namespaces=namespaces)
-        self.assertEquals(
+        self.assertEqual(
             namespaces["p"],
             "http://www.six-interbank-clearing.com/de/" "pain.001.001.03.ch.02.xsd",
         )
-        self.assertEquals(pay_method_xpath[0].text, "TRF")
+        self.assertEqual(pay_method_xpath[0].text, "TRF")
         sepa_xpath = xml_root.xpath(
             "//p:PmtInf/p:PmtTpInf/p:SvcLvl/p:Cd", namespaces=namespaces
         )
-        self.assertEquals(len(sepa_xpath), 0)
+        self.assertEqual(len(sepa_xpath), 0)
         local_instrument_xpath = xml_root.xpath(
             "//p:PmtInf/p:PmtTpInf/p:LclInstrm/p:Prtry", namespaces=namespaces
         )
-        self.assertEquals(len(local_instrument_xpath), 0)
+        self.assertEqual(len(local_instrument_xpath), 0)
 
         debtor_acc_xpath = xml_root.xpath(
             "//p:PmtInf/p:DbtrAcct/p:Id/p:IBAN", namespaces=namespaces
         )
-        self.assertEquals(
+        self.assertEqual(
             debtor_acc_xpath[0].text,
             self.payment_order.company_partner_bank_id.sanitized_acc_number,
         )
         self.payment_order.generated2uploaded()
-        self.assertEquals(self.payment_order.state, "uploaded")
+        self.assertEqual(self.payment_order.state, "uploaded")
         for inv in [invoice1, invoice2]:
-            self.assertEquals(inv.state, "paid")
+            self.assertEqual(inv.state, "posted")
```

Comment:
Same comment as similar line's in above test

Reply:
[https://github.com/OCA/l10n-switzerland/pull/593#discussion_r674862246](https://github.com/OCA/l10n-switzerland/pull/593#discussion_r674862246)